.github/workflows/cmake_build.yml

```
@@ -19,69 +19,71 @@ jobs:
-          cc: cl
-          cxx: cl
-          compiler_cache: sccache
-          compiler_cache_path: C:\Users\runneradmin\AppData\Local\Mozilla\sccache\cache
-          vcpkg_binary_cache: C:\Users\runneradmin\AppData\Local\vcpkg
-          vcpkg_triplet: x86-windows
-          exe_suffix: .exe
+        - name: macOS (Intel) - Catalina 10.15
+          os: macos-10.15
+          arch: Intel
+          vcpkg_triplet: x64-osx-10.12min
+          CMAKE_GENERATOR: Ninja
 
-        - name: Windows_64bit
-          os: windows-latest
+        - name: Windows (amd64) -  Windows Server 2019
+          os: windows-2019
           arch: amd64
-          # required so CMake uses MSVC rather than MinGW
-          cc: cl
-          cxx: cl
-          compiler_cache: sccache
-          compiler_cache_path: C:\Users\runneradmin\AppData\Local\Mozilla\sccache\cache
-          vcpkg_binary_cache: C:\Users\runneradmin\AppData\Local\vcpkg
           vcpkg_triplet: x64-windows
-          exe_suffix: .exe
+          CMAKE_GENERATOR: Visual Studio 16 2019
+          CMAKE_GENERATOR_PLATFORM: x64
+
+        - name: Windows (win32) - Windows Server 2019
+          os: windows-2019
+          arch: x86
+          vcpkg_triplet: x86-windows
+          CMAKE_GENERATOR: Visual Studio 16 2019
+          CMAKE_GENERATOR_PLATFORM: win32
+
 
     env:
-      CMAKE_BUILD_TYPE: RelWithDebInfo
-      CMAKE_C_COMPILER_LAUNCHER: ${{ matrix.config.os.compiler_cache }}
-      CMAKE_CXX_COMPILER_LAUNCHER: ${{ matrix.config.os.compiler_cache }}
+
+      AUDACITY_ARCH_LABEL: ${{ matrix.config.arch }}
       MACOSX_DEPLOYMENT_TARGET: 10.12
 
+      # CMake settings
+      CMAKE_BUILD_TYPE: RelWithDebInfo
+      CMAKE_GENERATOR: ${{matrix.config.CMAKE_GENERATOR}}
+      CMAKE_GENERATOR_PLATFORM: ${{matrix.config.CMAKE_GENERATOR_PLATFORM}}
+
+      # vcpkg settings
       VCPKG_DISABLE_METRICS: true
       VCPKG_DEFAULT_TRIPLET: ${{ matrix.config.vcpkg_triplet }}
       VCPKG_BINARY_SOURCES: "clear;default,readwrite;nuget,https://tenacityteam.jfrog.io/artifactory/api/nuget/tenacity-nuget,readwrite"
 
+      # Artifactory cache settings
+      JFROG_ARTIFACTORY_NUGET_USER: ${{secrets.JFROG_ARTIFACTORY_NUGET_USER}}
+      JFROG_ARTIFACTORY_NUGET_PASS: ${{secrets.JFROG_ARTIFACTORY_NUGET_PASS}}
+      JFROG_ARTIFACTORY_NUGET_TOKEN: ${{secrets.JFROG_ARTIFACTORY_NUGET_TOKEN}}
+
+      # Apple codesigning
+      APPLE_CODESIGN_IDENTITY: ${{ secrets.APPLE_CODESIGN_IDENTITY }}
+      APPLE_NOTARIZATION_USER_NAME: ${{ secrets.APPLE_NOTARIZATION_USER_NAME }}
+      APPLE_NOTARIZATION_PASSWORD: ${{ secrets.APPLE_NOTARIZATION_PASSWORD }}
+
       # Windows codesigning
-      # This variables will be used by all the steps
       WINDOWS_CERTIFICATE: ${{ secrets.WINDOWS_CERTIFICATE }}
       WINDOWS_CERTIFICATE_PASSWORD: ${{ secrets.WINDOWS_CERTIFICATE_PASSWORD }}
 
-      AUDACITY_ARCH_LABEL: ${{ matrix.config.arch }}
+      # WxWidgets info for linux builds
```

```suggestion
      # WxWidgets info for Linux and macOS builds
```
wxWidgets in Homebrew is built for a minimum macOS version of 10.15 so we're also building wxWidgets from source for macOS.